ci-operator/step-registry/baremetal/lab/install/agent/baremetal-lab-install-agent-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+RELEASE_IMAGE=$(oc adm release info --registry-config "${CLUSTER_PROFILE_DIR}/pull-secret" "${RELEASE_IMAGE_LATEST}" -o json|jq -r '.metadata.version')
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=INFO
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+ssh "${SSHOPTS[@]}" root@openshift-qe-055.arm.eng.rdu2.redhat.com <<EOF
+cp "/root/workdir/agent-bm-deployments/prow_${DEPLOYMENT_TYPE}" /root/workdir/agent-bm-deployments/prow_inventory
+cat <<OOO >> /root/workdir/agent-bm-deployments/prow_inventory
+DEPLOYMENT_TYPE=${DEPLOYMENT_TYPE}
+IP_STACK=${IP_STACK}
+CLUSTER_NAME=agent${DEPLOYMENT_TYPE}
+DISCONNECTED=${DISCONNECTED}
+PROXY=${PROXY}
+FIPS=${FIPS}
+RELEASE_IMAGE=registry.ci.openshift.org/ocp/release:${RELEASE_IMAGE}
+OOO
+
+cd /root/workdir/agent-bm-deployments/
+ansible-playbook -i prow_inventory install.yaml
+./openshift-install agent wait-for bootstrap-complete
```

Comment:
If we support multiple versions of agent install, `openshfit-install` need to change accordingly?

Reply:
we need multiple concurrent running jobs case, to avoid the openshift-installer is overridden by different jobs.

Reply:
Good point, let me take that into account.

Reply:
For the ansible-playbook running, seem like `openshift-qe-055.arm.eng.rdu2.redhat.com` is used for something like bastion, if it is, maybe we can prepare `prow_inventory` on slaves, and inject ssh proxy via `ansible_ssh_common_args` in the `prow_inventory` , even include `openshift-install agent` commands in the playbook together.